google-cloud-spanner/lib/google/cloud/spanner/database.rb

```diff
@@ -113,6 +113,14 @@ def state
           @grpc.state
         end
 
+        # An encryption configuration describing the encryption type and key
+        # resources in Cloud KMS.
+        #
+        # @return [Google::Cloud::Spanner::Admin::Database::V1::EncryptionConfig, nil]
+        def encryption_config
+          @grpc.encryption_config
+        end
+
```

Comment:
The CMEK change also included adding an `encryption_info` field to the `Database` message. See [here](https://github.com/googleapis/googleapis/commit/f829b1334cce86aa3738f3c0698d814b56664445#diff-9422939aee7fc2d7c95d6282558c7480d88ce72964608f9bf7c1c79dfe1fbb54R392-R403). Note that it is a repeated field and so will need to be treated as an array.

Reply:
Got it.